portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ["Education Level: Rising junior ", 'Age: 18 years old', 'DOB: November 27th, 2002', 'Height: 6ft'];
```

Comment:
Noticed "Education Level: Rising junior" uses double quotes, where the rest of the strings use single quotes. It might be good to update to be consistent in either direction.